modules/nw-ingress-configuring-application-domain.adoc

```diff
@@ -71,5 +73,5 @@ route.route.openshift.io/hello-openshift exposed
 ----
 $ oc get routes
 NAME              HOST/PORT                                   PATH   SERVICES          PORT       TERMINATION   WILDCARD
-hello-openshift   hello-openshift-my-project.apps.acme.io          hello-openshift   8080-tcp                 None
+hello-openshift   hello_openshift_my_project.apps.acme.io          hello-openshift   8080-tcp                 None
```

Comment:
The HOST/PORT of new created route should be like `hello-openshift-<my_project>.test.example.com`